testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -179,6 +179,8 @@
     "JaninArray", "LYSJaninArray", # time series of chi1/chi2 angles
     "PDB_rama", "PDB_janin", # for testing failures of Ramachandran and Janin classes
     "PDB_metal", # PDB with metal atoms
+    "Asph_gro"
+    "Asph_xtc"
```

Comment:
The error in the test is

```
from MDAnalysisTests.datafiles import *
E   AttributeError: module 'MDAnalysisTests.datafiles' has no attribute 'Asph_groAsph_xtcBATArray'
```

It happens when a module does `from datafiles import *`. In such cases, python looks for the `__all__` variable in the module. If it exists, python imports everything that is listed in there. The `__all__` variable contains an element `"Asph_groAsph_xtcBATArray" where you would want 3 distinct elements. This is because there is no coma between the 3 elements, so python takes them as a single string.

Try the following code:

```python
my_variable = ("a" "b" "c")  # see how there is no coma!
print(my_variable)  # prints abc
```

The solution to that problem is to add the missing comas.

Since you are at it, you could also add a comment to tell what the files are about.

Reply:
Thank you sooooo much. you are very helpful.